backend/danswer/server/documents/connector.py

```diff
@@ -559,7 +560,7 @@ def get_connector_indexing_status(
                 cc_pair_status=cc_pair.status,
                 connector=ConnectorSnapshot.from_connector_db_model(connector),
                 credential=CredentialSnapshot.from_credential_db_model(credential),
```

Comment:
access_type is now enum instead of boolean

Reply:
let's add a check at the api endpoint to throw an exception if they set it as sync but there's no implementation or they're on MIT

Reply:
maybe just if there's no implementation is fine, whatever is fast and fun